apps/api/e2e/tests/admin.test.ts

```diff
@@ -2358,4 +2359,173 @@ describe("Admin API", () => {
       (agentsAfterConversion as CompetitionAgentsResponse).agents,
     ).toHaveLength(2);
   });
+
+  // ===== Prize Pool Tests =====
+
+  test("should create competition with prizePools.agent and prizePools.users", async () => {
+    const client = createTestClient(getBaseUrl());
+    await client.loginAsAdmin(adminApiKey);
+
+    // Create a competition with prize pools
+    const createResponse = await axios.post(
+      `${getBaseUrl()}/api/admin/competition/create`,
```

Comment:
note: we have a test `ApiClient` that we should update to take these new params into account: [`createCompetition`](https://github.com/recallnet/js-recall/blob/f630a5eeb242f3fb9766c691978a962a1095980a/apps/api/e2e/utils/api-client.ts#L513) & [`updateCompetition`](https://github.com/recallnet/js-recall/blob/f630a5eeb242f3fb9766c691978a962a1095980a/apps/api/e2e/utils/api-client.ts#L589).

and then we can do:
```ts
const adminClient = createTestClient();
await adminClient.loginAsAdmin(adminApiKey);
const createCompetitionResult = await adminClient.createCompetition({
  ...
});
```